.template.config/template.json

```diff
@@ -3,8 +3,10 @@
 	"author": "stacks@amido.com",
 	"classifications": [
 		"Stacks",
+		"Application",
 		"Infrastructure",
 		"CQRS",
+		"Events",
 		"WebAPI"
 	],
 	"name": "Amido Stacks CQRS Web API",
```

Comment:
On line 15 of this file we need to change the name of the template. This is the equivalent of the `stacks-cqrs-events-app` and it contains not only the API, but the functions and background worker too. 


`stacks-cqrs-app` or `stacks-cqrs-webapp` both work for new name.